src/social/handles.ts

```diff
@@ -39,6 +42,22 @@ export const getUser = async (_id: Handle, _opts?: Config): Promise<User> => {
   throw NotImplementedError;
 };
 
+/**
+ * createUser() creates a new identity for a public key and registers a handle to it.
+ * This function will wait for the identity to land on chain before resolving.
+ * @param addr public key address that will be used to control identity delegate
+ * @param handle name of identity (must be globaly unique)
+ * @param opts Optional. Configuration overrides, such as from address, if any
+ * @return id of identity created
+ */
+export const createUser = async (addr: HexString, handle: Handle): Promise<number> => {
+  const txn = await createAndRegisterBeaconProxy(addr, handle);
+  const receipt = await txn.wait(1);
```

Comment:
could this be part of a config?

Reply:
This is a good point. There aren't going to be forks on the initial PoA network, so 1 is always sufficient. For that reason, I'd like to push off a configuration parameter until we have a better idea how config is going to be used.